### ISSUE TITLE
abnormal fix_subnet.json

### ISSUE THREAD

**Issue by wangwuwu** [open-mmlab/mmrazor#544]
### Checklist

- I have searched related issues but cannot get the expected help.
- I have read related documents and don't know what to do.

### Describe the question you meet
使用group fisher剪枝yolov8n,得到的sub_fixsub.json只含有stem和head
{
    "backbone.stem.conv_(0, 16)_16": 16,
    "bbox_head.head_module.cls_preds.0.0.conv_(0, 64)_64": 37,
    "bbox_head.head_module.cls_preds.0.1.conv_(0, 64)_64": 28,
    "bbox_head.head_module.reg_preds.0.0.conv_(0, 64)_64": 50,
    "bbox_head.head_module.reg_preds.0.1.conv_(0, 64)_64": 52,
    "bbox_head.head_module.cls_preds.1.0.conv_(0, 64)_64": 62,
    "bbox_head.head_module.cls_preds.1.1.conv_(0, 64)_64": 55,
    "bbox_head.head_module.reg_preds.1.0.conv_(0, 64)_64": 53,
    "bbox_head.head_module.reg_preds.1.1.conv_(0, 64)_64": 59,
    "backbone.stage4.2.conv1.conv_(0, 128)_128": 128,
    "bbox_head.head_module.cls_preds.2.0.conv_(0, 64)_64": 64,
    "bbox_head.head_module.cls_preds.2.1.conv_(0, 64)_64": 60,
    "bbox_head.head_module.reg_preds.2.0.conv_(0, 64)_64": 64,
    "bbox_head.head_module.reg_preds.2.1.conv_(0, 64)_64": 59
\[here\]

### Post related information

1. The output of `pip list | grep "mmcv\|mmrazor\|^torch"`
   \[here\]
2. Your config file if you modified it or created a new one.
interval = 10
normalization_type = 'act'
lr_ratio = 0.5

target_flop_ratio = 0.2
input_shape = (1, 3, 640, 640)
##############################################################################

architecture = _base_.model

if hasattr(_base_, 'data_preprocessor'):
    architecture.update({'data_preprocessor': _base_.data_preprocessor})
    data_preprocessor = {}

architecture.init_cfg = dict(type='Pretrained', checkpoint=pretrained_path)
architecture['_scope_'] = _base_.default_scope
architecture.backbone.frozen_stages = -1

model = dict(
    _delete_=True,
    _scope_='mmrazor',
    type='GroupFisherAlgorithm',
    architecture=architecture,
    interval=interval,
    mutator=dict(
        type='GroupFisherChannelMutator',
        parse_cfg=dict(type='ChannelAnalyzer', tracer_type='FxTracer'), # BackwardTracer
        channel_unit_cfg=dict(
            type='GroupFisherChannelUnit',
            default_args=dict(normalization_type=normalization_type, ),
        ),
    ),
)

model_wrapper_cfg = dict(
    type='mmrazor.GroupFisherDDP',
    broadcast_buffers=False,
)

optim_wrapper = dict(
    optimizer=dict(lr=_base_.optim_wrapper.optimizer.lr * lr_ratio))

custom_hooks = getattr(_base_, 'custom_hooks', []) + [
    dict(type='mmrazor.PruningStructureHook'),
    dict(
        type='mmrazor.ResourceInfoHook',
        interval=interval,
        demo_input=dict(
            type='mmrazor.DefaultDemoInput',
            input_shape=input_shape,
        ),
        save_ckpt_thr=[target_flop_ratio],
    ),
]

```python
[here]
```

3. Your train log file if you meet the problem during training.
   \[here\]
4. Other code you modified in the `mmrazor` folder.
   \[here\]
